content/docs/getting-started/runtime.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Runtime"
+weight : 1030
+menu:
+  docs:
+    parent: getting_started
+---
+
+In the previous section we explored Futures and Streams which allow us to represent
+a value (in the case of a `Future`) or a series of values (in the case of `Stream`)
+that will be available "at some point in the future". We talked about `poll` on `Future`
+and `Stream` which the runtime will call to figure out if the `Future` or the
+`Stream` are ready to yield a value.
+
+Lastly, we hinted at the idea of an exectuor which is the specific thing inside the
+runtime which polls Futures and Streams driving them to completion. We'll take a
+closer look at executors now.
+
+## Executors
```

Comment:
In the getting started part, maybe we shouldn't use the "executor" term and instead focus on "runtime".